backend/src/main/java/com/dhbw/tutorsystem/security/jwt/AuthEntryPointJwt.java

```diff
@@ -13,7 +13,7 @@
 @Component
 public class AuthEntryPointJwt implements AuthenticationEntryPoint {
 
-    private static final Logger logger = LoggerFactory.getLogger(AuthEntryPointJwt.class);
+    private static final Logger logger = LoggerFactory.getLogger("com");
```

Comment:
Bitte mit Robin abstimmen, warum das bisher so implementiert war

Reply:
Wenn das so implementiert ist, wie es bisher implementiert war wird einfach der root logger genommen, der alles in die Konsole schreibt. Spreche nochmal mit Robin